<!DOCTYPE html>
<html><head>
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
		<meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src data:; style-src 'unsafe-inline' data:">
		<title>Zotero Report</title>
		<link rel="stylesheet" type="text/css" href="data:text/css;base64,Ym9keSB7Cgljb2xvci1zY2hlbWU6IGxpZ2h0IGRhcms7CgkvKiBUaGVzZSBzaG91bGQgYmUgdGhlIGRlZmF1bHRzLCBidXQganVzdCBpbiBjYXNlOiAqLwoJYmFja2dyb3VuZDogQ2FudmFzOwoJY29sb3I6IENhbnZhc1RleHQ7Cn0KCmEgewoJdGV4dC1kZWNvcmF0aW9uOiB1bmRlcmxpbmU7Cn0KCmJvZHkgewoJcGFkZGluZzogMDsKfQoKdWwucmVwb3J0IGxpLml0ZW0gewoJYm9yZGVyLXRvcDogNHB4IHNvbGlkICM1NTU7CglwYWRkaW5nLXRvcDogMWVtOwoJcGFkZGluZy1sZWZ0OiAxZW07CglwYWRkaW5nLXJpZ2h0OiAxZW07CgltYXJnaW4tYm90dG9tOiAyZW07Cn0KCmgxLCBoMiwgaDMsIGg0LCBoNSwgaDYgewoJZm9udC13ZWlnaHQ6IG5vcm1hbDsKfQoKaDIgewoJbWFyZ2luOiAwIDAgLjVlbTsKfQoKaDIucGFyZW50SXRlbSB7Cglmb250LXdlaWdodDogNjAwOwoJZm9udC1zaXplOiAxZW07CglwYWRkaW5nOiAwIDAgLjVlbTsKCWJvcmRlci1ib3R0b206IDFweCBzb2xpZCAjY2NjOwp9CgovKiBJZiBjb21iaW5pbmcgY2hpbGRyZW4sIGRpc3BsYXkgcGFyZW50IHNsaWdodGx5IGxhcmdlciAqLwp1bC5yZXBvcnQuY29tYmluZUNoaWxkSXRlbXMgaDIucGFyZW50SXRlbSB7Cglmb250LXNpemU6IDEuMWVtOwoJcGFkZGluZy1ib3R0b206IC43NWVtOwoJbWFyZ2luLWJvdHRvbTogLjRlbTsKfQoKaDIucGFyZW50SXRlbSAudGl0bGUgewoJZm9udC13ZWlnaHQ6IG5vcm1hbDsKfQoKaDMgewoJbWFyZ2luLWJvdHRvbTogLjZlbTsKCWZvbnQtd2VpZ2h0OiA2MDAgIWltcG9ydGFudDsKCWZvbnQtc2l6ZTogMWVtOwoJZGlzcGxheTogYmxvY2s7Cn0KCi8qIE1ldGFkYXRhIHRhYmxlICovCnRoIHsKCXZlcnRpY2FsLWFsaWduOiB0b3A7Cgl0ZXh0LWFsaWduOiByaWdodDsKCXdpZHRoOiAxNSU7Cgl3aGl0ZS1zcGFjZTogbm93cmFwOwp9Cgp0ZCB7CglwYWRkaW5nLWxlZnQ6IC41ZW07Cn0KCgp1bC5yZXBvcnQsIHVsLm5vdGVzLCB1bC50YWdzIHsKCWxpc3Qtc3R5bGU6IG5vbmU7CgltYXJnaW4tbGVmdDogMDsKCXBhZGRpbmctbGVmdDogMDsKfQoKLyogVGFncyAqLwpoMy50YWdzIHsKCWZvbnQtc2l6ZTogMS4xZW07Cn0KCnVsLnRhZ3MgewoJbGluZS1oZWlnaHQ6IDEuNzVlbTsKCWxpc3Qtc3R5bGU6IG5vbmU7Cn0KCnVsLnRhZ3MgbGkgewoJZGlzcGxheTogaW5saW5lOwp9Cgp1bC50YWdzIGxpOm5vdCg6bGFzdC1jaGlsZCk6YWZ0ZXIgewoJY29udGVudDogJywgJzsKfQoKCi8qIENoaWxkIG5vdGVzICovCmgzLm5vdGVzIHsKCWZvbnQtc2l6ZTogMS4xZW07Cn0KCnVsLm5vdGVzIHsKCW1hcmdpbi1ib3R0b206IDEuMmVtOwp9Cgp1bC5ub3RlcyA+IGxpOmZpcnN0LWNoaWxkIHAgewoJbWFyZ2luLXRvcDogMDsKfQoKdWwubm90ZXMgPiBsaSB7CglwYWRkaW5nOiAuN2VtIDA7Cn0KCnVsLm5vdGVzID4gbGk6bm90KDpsYXN0LWNoaWxkKSB7Cglib3JkZXItYm90dG9tOiAxcHggI2NjYyBzb2xpZDsKfQoKCnVsLm5vdGVzID4gbGkgcDpmaXJzdC1jaGlsZCB7CgltYXJnaW4tdG9wOiAwOwp9Cgp1bC5ub3RlcyA+IGxpIHA6bGFzdC1jaGlsZCB7CgltYXJnaW4tYm90dG9tOiAwOwp9CgovKiBBZGQgcXVvdGF0aW9uIG1hcmtzIGFyb3VuZCBibG9ja3F1b3RlICovCnVsLm5vdGVzID4gbGkgYmxvY2txdW90ZSBwOm5vdCg6ZW1wdHkpOmJlZm9yZSwKbGkubm90ZSBibG9ja3F1b3RlIHA6bm90KDplbXB0eSk6YmVmb3JlIHsKCWNvbnRlbnQ6ICfigJwnOwp9Cgp1bC5ub3RlcyA+IGxpIGJsb2NrcXVvdGUgcDpub3QoOmVtcHR5KTpsYXN0LWNoaWxkOmFmdGVyLApsaS5ub3RlIGJsb2NrcXVvdGUgcDpub3QoOmVtcHR5KTpsYXN0LWNoaWxkOmFmdGVyIHsKCWNvbnRlbnQ6ICfigJ0nOwp9CgovKiBQcmVzZXJ2ZSB3aGl0ZXNwYWNlIG9uIHBsYWludGV4dCBub3RlcyAqLwp1bC5ub3RlcyBsaSBwLnBsYWludGV4dCwgbGkubm90ZSBwLnBsYWludGV4dCwgZGl2Lm5vdGUgcC5wbGFpbnRleHQgewoJd2hpdGUtc3BhY2U6IHByZS13cmFwOwp9CgovKiBEaXNwbGF5IHRhZ3Mgd2l0aGluIGNoaWxkIG5vdGVzIGlubGluZSAqLwp1bC5ub3RlcyBoMy50YWdzIHsKCWRpc3BsYXk6IGlubGluZTsKCWZvbnQtc2l6ZTogMWVtOwp9Cgp1bC5ub3RlcyBoMy50YWdzOmFmdGVyIHsKCWNvbnRlbnQ6ICcgJzsKfQoKdWwubm90ZXMgdWwudGFncyB7CglkaXNwbGF5OiBpbmxpbmU7Cn0KCnVsLm5vdGVzIHVsLnRhZ3MgbGk6bm90KDpsYXN0LWNoaWxkKTphZnRlciB7Cgljb250ZW50OiAnLCAnOwp9CgoKLyogQ2hpbGQgYXR0YWNobWVudHMgKi8KaDMuYXR0YWNobWVudHMgewoJZm9udC1zaXplOiAxLjFlbTsKfQoKdWwuYXR0YWNobWVudHMgbGkgewoJcGFkZGluZy10b3A6IC41ZW07Cn0KCnVsLmF0dGFjaG1lbnRzIGRpdi5ub3RlIHsKCW1hcmdpbi1sZWZ0OiAyZW07Cn0KCnVsLmF0dGFjaG1lbnRzIGRpdi5ub3RlIHA6Zmlyc3QtY2hpbGQgewoJbWFyZ2luLXRvcDogLjc1ZW07Cn0KCmRpdiB0YWJsZSB7Cglib3JkZXItY29sbGFwc2U6IGNvbGxhcHNlOwp9CgpkaXYgdGFibGUgdGQsIGRpdiB0YWJsZSB0aCB7Cglib3JkZXI6IDFweCAjY2NjIHNvbGlkOwoJYm9yZGVyLWNvbGxhcHNlOiBjb2xsYXBzZTsKCXdvcmQtYnJlYWs6IGJyZWFrLWFsbDsKfQoKZGl2IHRhYmxlIHRkIHA6ZW1wdHk6OmFmdGVyLCBkaXYgdGFibGUgdGggcDplbXB0eTo6YWZ0ZXIgewoJY29udGVudDogIlwwMGEwIjsKfQoKZGl2IHRhYmxlIHRkICo6Zmlyc3QtY2hpbGQsIGRpdiB0YWJsZSB0aCAqOmZpcnN0LWNoaWxkIHsKCW1hcmdpbi10b3A6IDA7Cn0KCmRpdiB0YWJsZSB0ZCAqOmxhc3QtY2hpbGQsIGRpdiB0YWJsZSB0aCAqOmxhc3QtY2hpbGQgewoJbWFyZ2luLWJvdHRvbTogMDsKfQo=">
		<link rel="stylesheet" type="text/css" media="screen,projection" href="data:text/css;base64,LyogR2VuZXJpYyBzdHlsZXMgKi8KYm9keSB7Cglmb250OiA2Mi41JSBHZW9yZ2lhLCBUaW1lcywgc2VyaWY7Cgl3aWR0aDogNzgwcHg7CgltYXJnaW46IDAgYXV0bzsKfQoKaDIgewoJZm9udC1zaXplOiAxLjVlbTsKCWxpbmUtaGVpZ2h0OiAxLjVlbTsKCWZvbnQtZmFtaWx5OiBHZW9yZ2lhLCBUaW1lcywgc2VyaWY7Cn0KCnAgewoJbGluZS1oZWlnaHQ6IDEuNWVtOwp9CgphOmFueS1saW5rIHsKCWNvbG9yOiAjOTAwOwp9CgphOmhvdmVyLCBhOmFjdGl2ZSB7Cgljb2xvcjogIzc3NzsKfQoKQG1lZGlhIChwcmVmZXJzLWNvbG9yLXNjaGVtZTogZGFyaykgewoJYTphbnktbGluayB7CgkJY29sb3I6ICNmMDA7Cgl9CgoJYTpob3ZlciwgYTphY3RpdmUgewoJCWNvbG9yOiAjOTk5OwoJfQp9CgoKdWwucmVwb3J0IHsKCWZvbnQtc2l6ZTogMS40ZW07Cgl3aWR0aDogNjgwcHg7CgltYXJnaW46IDAgYXV0bzsKCXBhZGRpbmc6IDIwcHggMjBweDsKfQoKLyogTWV0YWRhdGEgdGFibGUgKi8KdGFibGUgewoJYm9yZGVyOiAxcHggI2NjYyBzb2xpZDsKCW92ZXJmbG93OiBhdXRvOwoJd2lkdGg6IDEwMCU7CgltYXJnaW46IC4xZW0gYXV0byAuNzVlbTsKCXBhZGRpbmc6IDAuNWVtOwp9Cg==">
		<link rel="stylesheet" type="text/css" media="print" href="data:text/css;base64,Ym9keSB7Cglmb250OiAxMnB0ICJUaW1lcyBOZXcgUm9tYW4iLCBUaW1lcywgR2VvcmdpYSwgc2VyaWY7CgltYXJnaW46IDA7Cgl3aWR0aDogYXV0bzsKfQoKLyogUGFnZSBCcmVha3MgKHBhZ2UtYnJlYWstaW5zaWRlIG9ubHkgcmVjb2duaXplZCBieSBPcGVyYSkgKi8KaDEsIGgyLCBoMywgaDQsIGg1LCBoNiB7CglwYWdlLWJyZWFrLWFmdGVyOiBhdm9pZDsKCXBhZ2UtYnJlYWstaW5zaWRlOiBhdm9pZDsKfQoKdWwsIG9sLCBkbCB7CglwYWdlLWJyZWFrLWluc2lkZTogYXZvaWQ7Cgljb2xvci1hZGp1c3Q6IGV4YWN0Owp9CgpoMiB7Cglmb250LXNpemU6IDEuM2VtOwoJbGluZS1oZWlnaHQ6IDEuM2VtOwp9CgphIHsKCWNvbG9yOiBpbmhlcml0OwoJdGV4dC1kZWNvcmF0aW9uOiBub25lOwp9Cg==">
	</head>
	<body>
		<ul class="report combineChildItems">
			<li id="item_LRK6GYCX" class="item computerProgram">
			<h2>BiocManager: Access the Bioconductor Project Package Repository</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Martin Morgan</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Marcel Ramos</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2025</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://cran.r-project.org/package=BiocManager">https://CRAN.R-project.org/package=BiocManager</a></td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>DOI: 10.32614/CRAN.package.BiocManager</td>
					</tr>
					<tr>
					<th>Version</th>
						<td>1.30.26</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:34:24 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:45:37 AM</td>
					</tr>
				</tbody></table>
				<h3 class="notes">Notes:</h3>
				<ul class="notes">
					<li id="item_YNQZTSXG">
<div><p>R package version 1.30.26</p></div>
					</li>
				</ul>
			</li>


			<li id="item_7JLXRHSV" class="item journalArticle">
			<h2>clusterMaker: a multi-algorithm clustering plugin for Cytoscape</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>John H. Morris</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Leonard Apeltsin</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Aaron M. Newman</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Jan Baumbach</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Tobias Wittkop</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Gang Su</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Gary D. Bader</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Thomas E. Ferrin</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>BACKGROUND: In the post-genomic era, the rapid increase in 
high-throughput data calls for computational tools capable of 
integrating data of diverse types and facilitating recognition of 
biologically meaningful patterns within them. For example, 
protein-protein interaction data sets have been clustered to identify 
stable complexes, but scientists lack easily accessible tools to 
facilitate combined analyses of multiple data sets from different types 
of experiments. Here we present clusterMaker, a Cytoscape plugin that 
implements several clustering algorithms and provides network, 
dendrogram, and heat map views of the results. The Cytoscape network is 
linked to all of the other views, so that a selection in one is 
immediately reflected in the others. clusterMaker is the first Cytoscape
 plugin to implement such a wide variety of clustering algorithms and 
visualizations, including the only implementations of hierarchical 
clustering, dendrogram plus heat map visualization (tree view), k-means,
 k-medoid, SCPS, AutoSOME, and native (Java) MCL.
RESULTS: Results are presented in the form of three scenarios of use: 
analysis of protein expression data using a recently published mouse 
interactome and a mouse microarray data set of nearly one hundred 
diverse cell/tissue types; the identification of protein complexes in 
the yeast Saccharomyces cerevisiae; and the cluster analysis of the 
vicinal oxygen chelate (VOC) enzyme superfamily. For scenario one, we 
explore functionally enriched mouse interactomes specific to particular 
cellular phenotypes and apply fuzzy clustering. For scenario two, we 
explore the prefoldin complex in detail using both physical and genetic 
interaction clusters. For scenario three, we explore the possible 
annotation of a protein as a methylmalonyl-CoA epimerase within the VOC 
superfamily. Cytoscape session files for all three scenarios are 
provided in the Additional Files section.
CONCLUSIONS: The Cytoscape plugin clusterMaker provides a number of 
clustering algorithms and visualizations that can be used independently 
or in combination for analysis and visualization of biological data 
sets, and for confirming or generating hypotheses about biological 
function. Several of these visualizations and algorithms are only 
available to Cytoscape users through the clusterMaker plugin. 
clusterMaker is available via the Cytoscape plugin manager.</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2011-11-09</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>eng</td>
					</tr>
					<tr>
					<th>Short Title</th>
						<td>clusterMaker</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>PubMed</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>PMID: 22070249
PMCID: PMC3262844</td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>12</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>436</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>BMC bioinformatics</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1186/1471-2105-12-436">10.1186/1471-2105-12-436</a></td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>BMC Bioinformatics</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1471-2105</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/29/2025, 11:56:34 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/29/2025, 11:56:34 AM</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>Animals</li>
					<li>Genomics</li>
					<li>Software</li>
					<li>Algorithms</li>
					<li>Cluster Analysis</li>
					<li>Mice</li>
					<li>Protein Interaction Maps</li>
					<li>Racemases and Epimerases</li>
					<li>Saccharomyces cerevisiae</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_S2JLR5V5">Full Text					</li>
					<li id="item_6VBGXGUC">PubMed entry					</li>
				</ul>
			</li>


			<li id="item_9FXUH2UR" class="item journalArticle">
			<h2>Computing topological parameters of biological networks</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Yassen Assenov</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Fidel Ramírez</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Sven-Eric Schelhorn</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Thomas Lengauer</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Mario Albrecht</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Rapidly increasing amounts of molecular interaction data are 
being produced by various experimental techniques and computational 
prediction methods. In order to gain insight into the organization and 
structure of the resultant large complex networks formed by the 
interacting molecules, we have developed the versatile Cytoscape plugin 
NetworkAnalyzer. It computes and displays a comprehensive set of 
topological parameters, which includes the number of nodes, edges, and 
connected components, the network diameter, radius, density, 
centralization, heterogeneity, and clustering coefficient, the 
characteristic path length, and the distributions of node degrees, 
neighborhood connectivities, average clustering coefficients, and 
shortest path lengths. NetworkAnalyzer can be applied to both directed 
and undirected networks and also contains extra functionality to 
construct the intersection or union of two networks. It is an 
interactive and highly customizable application that requires no expert 
knowledge in graph theory from the user.
AVAILABILITY: NetworkAnalyzer can be downloaded via the Cytoscape web 
site: http://www.cytoscape.org</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2008-01-15</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>eng</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>PubMed</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>PMID: 18006545</td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>24</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>282-284</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Bioinformatics (Oxford, England)</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1093/bioinformatics/btm554">10.1093/bioinformatics/btm554</a></td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>2</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>Bioinformatics</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1367-4811</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/29/2025, 11:56:21 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/29/2025, 11:56:21 AM</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>Signal Transduction</li>
					<li>Algorithms</li>
					<li>Protein Interaction Mapping</li>
					<li>User-Computer Interface</li>
					<li>Models, Biological</li>
					<li>Computer Simulation</li>
					<li>Proteome</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_ADBL2GEF">Full Text PDF					</li>
					<li id="item_TNZMBWS2">PubMed entry					</li>
				</ul>
			</li>


			<li id="item_LQASNQYL" class="item journalArticle">
			<h2>CyTargetLinker app update: A flexible solution for network extension in Cytoscape</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Martina Kutmon</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Friederike Ehrhart</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Egon L. Willighagen</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chris T. Evelo</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Susan L. Coort</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Here, we present an update of the open-source CyTargetLinker 
app for Cytoscape ( http://apps.cytoscape.org/apps/cytargetlinker) that 
introduces new automation features. CyTargetLinker provides a simple 
interface to extend networks with links to relevant data and/or 
knowledge extracted from so-called linksets. The linksets are provided 
on the CyTargetLinker website ( https://cytargetlinker.github.io/) or 
can be custom-made for specific use cases. The new automation feature 
enables users to programmatically execute the app's functionality in 
Cytoscape (command line tool) and with external tools (e.g. R, Jupyter, 
Python, etc). This allows users to share their analysis workflows and 
therefore increase repeatability and reproducibility. Three use cases 
demonstrate automated workflows, combinations with other Cytoscape apps 
and core Cytoscape functionality. We first extend a protein-protein 
interaction network created with the stringApp, with compound-target 
interactions and disease-gene annotations. In the second use case, we 
created a workflow to load differentially expressed genes from an 
experimental dataset and extend it with gene-pathway associations. 
Lastly, we chose an example outside the biological domain and used 
CyTargetLinker to create an author-article-journal network for the five 
authors of this manuscript using a two-step extension mechanism. With 
400 downloads per month in the last year and nearly 20,000 downloads in 
total, CyTargetLinker shows the adoption and relevance of the app in the
 field of network biology. In August 2019, the original publication was 
cited in 83 articles demonstrating the applicability in biomedical 
research.</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2018</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>eng</td>
					</tr>
					<tr>
					<th>Short Title</th>
						<td>CyTargetLinker app update</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>PubMed</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>PMID: 31489175
PMCID: PMC6707396</td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>7</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>ELIXIR-743</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>F1000Research</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.12688/f1000research.14613.2">10.12688/f1000research.14613.2</a></td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>F1000Res</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>2046-1402</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/29/2025, 11:56:08 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/29/2025, 11:56:08 AM</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>Software</li>
					<li>CyTargetLinker</li>
					<li>Cytoscape</li>
					<li>data integration</li>
					<li>Molecular Sequence Annotation</li>
					<li>network extension</li>
					<li>network visualization</li>
					<li>regulatory networks</li>
					<li>Reproducibility of Results</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_B2BQ22RK">Full Text					</li>
					<li id="item_KVBQTNNW">PubMed entry					</li>
				</ul>
			</li>


			<li id="item_F3ZZV6SC" class="item journalArticle">
			<h2>Cytoscape StringApp: Network Analysis and Visualization of Proteomics Data</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Nadezhda T. Doncheva</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>John H. Morris</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Jan Gorodkin</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Lars J. Jensen</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Protein networks have become a popular tool for analyzing and 
visualizing the often long lists of proteins or genes obtained from 
proteomics and other high-throughput technologies. One of the most 
popular sources of such networks is the STRING database, which provides 
protein networks for more than 2000 organisms, including both physical 
interactions from experimental data and functional associations from 
curated pathways, automatic text mining, and prediction methods. 
However, its web interface is mainly intended for inspection of small 
networks and their underlying evidence. The Cytoscape software, on the 
other hand, is much better suited for working with large networks and 
offers greater flexibility in terms of network analysis, import, and 
visualization of additional data. To include both resources in the same 
workflow, we created stringApp, a Cytoscape app that makes it easy to 
import STRING networks into Cytoscape, retains the appearance and many 
of the features of STRING, and integrates data from associated 
databases. Here, we introduce many of the stringApp features and show 
how they can be used to carry out complex network analysis and 
visualization tasks on a typical proteomics data set, all through the 
Cytoscape user interface. stringApp is freely available from the 
Cytoscape app store: http://apps.cytoscape.org/apps/stringapp .</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2019-02-01</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>eng</td>
					</tr>
					<tr>
					<th>Short Title</th>
						<td>Cytoscape StringApp</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>PubMed</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>PMID: 30450911
PMCID: PMC6800166</td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>18</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>623-632</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Journal of Proteome Research</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1021/acs.jproteome.8b00702">10.1021/acs.jproteome.8b00702</a></td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>2</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>J Proteome Res</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1535-3907</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/29/2025, 11:56:50 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/29/2025, 11:56:50 AM</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>Proteomics</li>
					<li>Software</li>
					<li>User-Computer Interface</li>
					<li>Computational Biology</li>
					<li>Internet</li>
					<li>Cytoscape</li>
					<li>network visualization</li>
					<li>Protein Interaction Maps</li>
					<li>Data Analysis</li>
					<li>functional enrichment</li>
					<li>network analysis</li>
					<li>protein networks</li>
					<li>proteomics data</li>
					<li>STRING database</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_5L3SHFX5">Accepted Version					</li>
					<li id="item_B54D76NY">PubMed entry					</li>
				</ul>
			</li>


			<li id="item_JIHNNRLL" class="item journalArticle">
			<h2>Cytoscape: a software environment for integrated models of biomolecular interaction networks</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Paul Shannon</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Andrew Markiel</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Owen Ozier</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Nitin S. Baliga</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Jonathan T. Wang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Daniel Ramage</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Nada Amin</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Benno Schwikowski</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Trey Ideker</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Cytoscape is an open source software project for integrating 
biomolecular interaction networks with high-throughput expression data 
and other molecular states into a unified conceptual framework. Although
 applicable to any system of molecular components and interactions, 
Cytoscape is most powerful when used in conjunction with large databases
 of protein-protein, protein-DNA, and genetic interactions that are 
increasingly available for humans and model organisms. Cytoscape's 
software Core provides basic functionality to layout and query the 
network; to visually integrate the network with expression profiles, 
phenotypes, and other molecular states; and to link the network to 
databases of functional annotations. The Core is extensible through a 
straightforward plug-in architecture, allowing rapid development of 
additional computational analyses and features. Several case studies of 
Cytoscape plug-ins are surveyed, including a search for interaction 
pathways correlating with changes in gene expression, a study of protein
 complexes involved in cellular recovery to DNA damage, inference of a 
combined physical/functional interaction network for Halobacterium, and 
an interface to detailed stochastic/kinetic gene regulatory models.</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2003-11</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>eng</td>
					</tr>
					<tr>
					<th>Short Title</th>
						<td>Cytoscape</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>PubMed</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>PMID: 14597658
PMCID: PMC403769</td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>13</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>2498-2504</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Genome Research</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1101/gr.1239303">10.1101/gr.1239303</a></td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>11</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>Genome Res</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1088-9051</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/29/2025, 11:55:53 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/29/2025, 9:33:43 PM</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>Phenotype</li>
					<li>Software</li>
					<li>Algorithms</li>
					<li>Archaeal Proteins</li>
					<li>Bacteriophage lambda</li>
					<li>Computational Biology</li>
					<li>Halobacterium</li>
					<li>Internet</li>
					<li>Models, Biological</li>
					<li>Neural Networks, Computer</li>
					<li>Software Design</li>
					<li>Stochastic Processes</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_JAVRAC68">Full Text					</li>
					<li id="item_IT7DZSRI">PubMed entry					</li>
				</ul>
			</li>


			<li id="item_GAN8U7QG" class="item computerProgram">
			<h2>dplyr: A Grammar of Data Manipulation</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Hadley Wickham</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Romain François</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Lionel Henry</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Kirill Müller</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Davis Vaughan</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2023</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://cran.r-project.org/package=dplyr">https://CRAN.R-project.org/package=dplyr</a></td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>DOI: 10.32614/CRAN.package.dplyr</td>
					</tr>
					<tr>
					<th>Version</th>
						<td>1.1.4</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:38:21 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:46:28 AM</td>
					</tr>
				</tbody></table>
				<h3 class="notes">Notes:</h3>
				<ul class="notes">
					<li id="item_ZT5MWPUW">
<div><p>R package version 1.1.4</p></div>
					</li>
				</ul>
			</li>


			<li id="item_ZND8WBR7" class="item journalArticle">
			<h2>enhancedGraphics: a Cytoscape app for enhanced node graphics</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>John H. Morris</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Allan Kuchinsky</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Thomas E. Ferrin</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Alexander R. Pico</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>enhancedGraphics ( 
http://apps.cytoscape.org/apps/enhancedGraphics) is a Cytoscape app that
 implements a series of enhanced charts and graphics that may be added 
to Cytoscape nodes. It enables users and other app developers to create 
pie, line, bar, and circle plots that are driven by columns in the 
Cytoscape Node Table. Charts are drawn using vector graphics to allow 
full-resolution scaling.</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2014</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>eng</td>
					</tr>
					<tr>
					<th>Short Title</th>
						<td>enhancedGraphics</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>PubMed</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>PMID: 25285206
PMCID: PMC4176421</td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>3</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>147</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>F1000Research</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.12688/f1000research.4460.1">10.12688/f1000research.4460.1</a></td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>F1000Res</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>2046-1402</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/29/2025, 11:56:42 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/29/2025, 11:56:42 AM</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_VFNL4ZQG">Full Text					</li>
					<li id="item_C5NMTDJS">PubMed entry					</li>
				</ul>
			</li>


			<li id="item_FGH2LVSL" class="item computerProgram">
			<h2>EnhancedVolcano: Publication-ready volcano plots with enhanced colouring and labeling</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Kevin Blighe</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Sharmila Rana</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Myles Lewis</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2025</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://bioconductor.org/packages/EnhancedVolcano">https://bioconductor.org/packages/EnhancedVolcano</a></td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>DOI: 10.18129/B9.bioc.EnhancedVolcano</td>
					</tr>
					<tr>
					<th>Version</th>
						<td>1.26.0</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:38:42 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:46:14 AM</td>
					</tr>
				</tbody></table>
				<h3 class="notes">Notes:</h3>
				<ul class="notes">
					<li id="item_8KV5G2BU">
<div><p>R package version 1.26.0</p></div>
					</li>
				</ul>
			</li>


			<li id="item_6PVZ4WGV" class="item computerProgram">
			<h2>enrichplot: Visualization of Functional Enrichment Result</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Guangchuang Yu</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2025</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://bioconductor.org/packages/enrichplot">https://bioconductor.org/packages/enrichplot</a></td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>DOI: 10.18129/B9.bioc.enrichplot</td>
					</tr>
					<tr>
					<th>Version</th>
						<td>1.28.2</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:40:53 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:45:18 AM</td>
					</tr>
				</tbody></table>
				<h3 class="notes">Notes:</h3>
				<ul class="notes">
					<li id="item_MUXUI4IF">
<div><p>R package version 1.28.2</p></div>
					</li>
				</ul>
			</li>


			<li id="item_E7RWNPLF" class="item journalArticle">
			<h2>g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update)</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Uku Raudvere</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Liis Kolberg</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Ivan Kuzmin</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Tambet Arak</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Priit Adler</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Hedi Peterson</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Jaak Vilo</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Biological data analysis often deals with lists of genes 
arising from various studies. The g:Profiler toolset is widely used for 
finding biological categories enriched in gene lists, conversions 
between gene identifiers and mappings to their orthologs. The mission of
 g:Profiler is to provide a reliable service based on up-to-date high 
quality data in a convenient manner across many evidence types, 
identifier spaces and organisms. g:Profiler relies on Ensembl as a 
primary data source and follows their quarterly release cycle while 
updating the other data sources simultaneously. The current update 
provides a better user experience due to a modern responsive web 
interface, standardised API and libraries. The results are delivered 
through an interactive and configurable web design. Results can be 
downloaded as publication ready visualisations or delimited text files. 
In the current update we have extended the support to 467 species and 
strains, including vertebrates, plants, fungi, insects and parasites. By
 supporting user uploaded custom GMT files, g:Profiler is now capable of
 analysing data from any organism. All past releases are maintained for 
reproducibility and transparency. The 2019 update introduces an 
extensive technical rewrite making the services faster and more 
flexible. g:Profiler is freely available at 
https://biit.cs.ut.ee/gprofiler.</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2019-07-02</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>eng</td>
					</tr>
					<tr>
					<th>Short Title</th>
						<td>g</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>PubMed</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>PMID: 31066453
PMCID: PMC6602461</td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>47</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>W191-W198</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Nucleic Acids Research</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1093/nar/gkz369">10.1093/nar/gkz369</a></td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>W1</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>Nucleic Acids Res</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1362-4962</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/29/2025, 11:56:14 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/29/2025, 11:56:14 AM</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>Humans</li>
					<li>Animals</li>
					<li>Databases, Genetic</li>
					<li>Software</li>
					<li>Fungi</li>
					<li>Genome</li>
					<li>Information Storage and Retrieval</li>
					<li>Parasites</li>
					<li>Plants</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_7UHNBART">Full Text					</li>
					<li id="item_3TNH6CJ5">PubMed entry					</li>
				</ul>
			</li>


			<li id="item_U4RFLM84" class="item computerProgram">
			<h2>msigdbr: MSigDB Gene Sets for Multiple Organisms in a Tidy Data Format</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Igor Dolgalev</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2025</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://cran.r-project.org/package=msigdbr">https://CRAN.R-project.org/package=msigdbr</a></td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>DOI: 10.32614/CRAN.package.msigdbr</td>
					</tr>
					<tr>
					<th>Version</th>
						<td>24.1.0</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:41:52 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:45:58 AM</td>
					</tr>
				</tbody></table>
				<h3 class="notes">Notes:</h3>
				<ul class="notes">
					<li id="item_IY7IAX5C">
<div><p>R package version 24.1.0</p></div>
					</li>
				</ul>
			</li>


			<li id="item_DJDIB55N" class="item computerProgram">
			<h2>org.Hs.eg.db</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Marc Carlson</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2025</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>DOI.org (Datacite)</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://bioconductor.org/packages/org.Hs.eg.db">https://bioconductor.org/packages/org.Hs.eg.db</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>6/25/2025, 11:35:12 AM</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>DOI: 10.18129/B9.BIOC.ORG.HS.EG.DB</td>
					</tr>
					<tr>
					<th>Version</th>
						<td>3.21.0</td>
					</tr>
					<tr>
					<th>Company</th>
						<td>Bioconductor</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:35:12 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:36:56 AM</td>
					</tr>
				</tbody></table>
			</li>


			<li id="item_9248CJE9" class="item computerProgram">
			<h2>Peiprjs/TCGA-CDK-analysis</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Mar Roca Cugat</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Ina De Rijk</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Ivan Sapsai</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Daniela Sara Mirensky Roffe</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2025-06-29T10:01:16Z</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>GitHub</td>
					</tr>
					<tr>
					<th>URL</th>
						<td>doi.org/10.5281/zenodo.15767006</td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>6/23/2025, 2:24:32 PM</td>
					</tr>
					<tr>
					<th>Rights</th>
						<td>AGPL-3.0</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>original-date: 2025-06-22T23:25:44Z</td>
					</tr>
					<tr>
					<th>Prog. Language</th>
						<td>R</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/23/2025, 2:24:32 PM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/29/2025, 12:26:46 PM</td>
					</tr>
				</tbody></table>
			</li>


			<li id="item_FI22JQVK" class="item computerProgram">
			<h2>R: A Language and Environment for Statistical Computing</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>R Core Team</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2025</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://www.r-project.org/">https://www.R-project.org/</a></td>
					</tr>
					<tr>
					<th>Place</th>
						<td>Vienna, Austria</td>
					</tr>
					<tr>
					<th>Version</th>
						<td>4.5.1</td>
					</tr>
					<tr>
					<th>Company</th>
						<td>R Foundation for Statistical Computing</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:32:55 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:35:54 AM</td>
					</tr>
				</tbody></table>
			</li>


			<li id="item_IJ3NP85L" class="item computerProgram">
			<h2>RColorBrewer: ColorBrewer Palettes</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Erich Neuwirth</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2022</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://cran.r-project.org/package=RColorBrewer">https://CRAN.R-project.org/package=RColorBrewer</a></td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>DOI: 10.32614/CRAN.package.RColorBrewer</td>
					</tr>
					<tr>
					<th>Version</th>
						<td>1.1-3</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:42:07 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:46:35 AM</td>
					</tr>
				</tbody></table>
				<h3 class="notes">Notes:</h3>
				<ul class="notes">
					<li id="item_LDD4H3X3">
<div><p>R package version 1.1-3</p></div>
					</li>
				</ul>
			</li>


			<li id="item_HLNYZ39H" class="item journalArticle">
			<h2>RCy3: Network Biology using Cytoscape from within R</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Gustavsen</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Julia A</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Pai</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Shraddha</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Isserlin</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Ruth</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Demchak</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Barry</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Pico</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Alexander R</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2019</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>F1000Research</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.12688/f1000research.20887.3">10.12688/f1000research.20887.3</a></td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:41:22 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:41:22 AM</td>
					</tr>
				</tbody></table>
			</li>


			<li id="item_WFBQ96QJ" class="item computerProgram">
			<h2>readr: Read Rectangular Text Data</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Hadley Wickham</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Jim Hester</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Jennifer Bryan</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2024</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://cran.r-project.org/package=readr">https://CRAN.R-project.org/package=readr</a></td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>DOI: 10.32614/CRAN.package.readr</td>
					</tr>
					<tr>
					<th>Version</th>
						<td>2.1.5</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:43:44 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:46:21 AM</td>
					</tr>
				</tbody></table>
				<h3 class="notes">Notes:</h3>
				<ul class="notes">
					<li id="item_4A9GJ5DT">
<div><p>R package version 2.1.5</p></div>
					</li>
				</ul>
			</li>


			<li id="item_9SF77GAT" class="item computerProgram">
			<h2>readxl: Read Excel Files</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Hadley Wickham</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Jennifer Bryan</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2025</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://cran.r-project.org/package=readxl">https://CRAN.R-project.org/package=readxl</a></td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>DOI: 10.32614/CRAN.package.readxl</td>
					</tr>
					<tr>
					<th>Version</th>
						<td>1.4.5</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:39:03 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:45:24 AM</td>
					</tr>
				</tbody></table>
				<h3 class="notes">Notes:</h3>
				<ul class="notes">
					<li id="item_KYW2MXDR">
<div><p>R package version 1.4.5</p></div>
					</li>
				</ul>
			</li>


			<li id="item_IFYEMBR3" class="item computerProgram">
			<h2>Rgraphviz: Provides plotting capabilities for R graph objects</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Kasper Daniel Hansen</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Jeff Gentry</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Li Long</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Robert Gentleman</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Seth Falcon</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Florian Hahne</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Deepayan Sarkar</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2025</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://bioconductor.org/packages/Rgraphviz">https://bioconductor.org/packages/Rgraphviz</a></td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>DOI: 10.18129/B9.bioc.Rgraphviz</td>
					</tr>
					<tr>
					<th>Version</th>
						<td>2.52.0</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:41:06 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:45:46 AM</td>
					</tr>
				</tbody></table>
				<h3 class="notes">Notes:</h3>
				<ul class="notes">
					<li id="item_WM3N5VMM">
<div><p>R package version 2.52.0</p></div>
					</li>
				</ul>
			</li>


			<li id="item_QYMSWP6M" class="item computerProgram">
			<h2>RStudio: Integrated Development Environment for R</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Software</td>
					</tr>
					<tr>
						<th class="programmer">Programmer</th>
						<td>Posit team</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2025</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://www.posit.co/">http://www.posit.co/</a></td>
					</tr>
					<tr>
					<th>Place</th>
						<td>Boston, MA</td>
					</tr>
					<tr>
					<th>Version</th>
						<td>2025.5.1.513</td>
					</tr>
					<tr>
					<th>Company</th>
						<td>Posit Software, PBC</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:32:18 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:35:50 AM</td>
					</tr>
				</tbody></table>
			</li>


			<li id="item_MFMXNFCZ" class="item journalArticle">
			<h2>Using clusterProfiler to characterize multiomics data</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Shuangbin Xu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Erqiang Hu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Yantong Cai</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Zijing Xie</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Xiao Luo</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Li Zhan</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Wenli Tang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Qianwen Wang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Bingdong Liu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Rui Wang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Wenqin Xie</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Tianzhi Wu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Liwei Xie</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Guangchuang Yu</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2024-11</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://www.nature.com/articles/s41596-024-01020-z">https://www.nature.com/articles/s41596-024-01020-z</a></td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>19</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>3292-3320</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Nature Protocols</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1038/s41596-024-01020-z">10.1038/s41596-024-01020-z</a></td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>11</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:40:33 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:40:33 AM</td>
					</tr>
				</tbody></table>
			</li>


			<li id="item_73AJZZLA" class="item journalArticle">
			<h2>WikiPathways App for Cytoscape: Making biological pathways amenable to network analysis and visualization</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Martina Kutmon</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Samad Lotia</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chris T. Evelo</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Alexander R. Pico</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>In this paper we present the open-source WikiPathways app for 
Cytoscape ( http://apps.cytoscape.org/apps/wikipathways) that can be 
used to import biological pathways for data visualization and network 
analysis. WikiPathways is an open, collaborative biological pathway 
database that provides fully annotated pathway diagrams for manual 
download or through web services. The WikiPathways app allows users to 
load pathways in two different views: as an annotated pathway ideal for 
data visualization and as a simple network to perform computational 
analysis. An example pathway and dataset are used to demonstrate the 
functionality of the WikiPathways app and how they can be combined and 
used together with other apps. More than 3000 downloads in the first 12 
months following its release in August 2013 highlight the importance and
 adoption of the app in the network biology field.</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2014</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>eng</td>
					</tr>
					<tr>
					<th>Short Title</th>
						<td>WikiPathways App for Cytoscape</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>PubMed</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>PMID: 25254103
PMCID: PMC4168754</td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>3</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>152</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>F1000Research</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.12688/f1000research.4254.2">10.12688/f1000research.4254.2</a></td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>F1000Res</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>2046-1402</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/29/2025, 11:56:28 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/29/2025, 11:56:28 AM</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_M2GEL62S">Full Text					</li>
					<li id="item_GIT4L49M">PubMed entry					</li>
				</ul>
			</li>


			<li id="item_AV3KYHPT" class="item journalArticle">
			<h2>WikiPathways: a multifaceted pathway database bridging metabolomics to other omics research</h2>
				<table>
					<tbody><tr>
						<th>Item Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Denise N. Slenter</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Martina Kutmon</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Kristina Hanspers</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Anders Riutta</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Jacob Windsor</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Nuno Nunes</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Jonathan Mélius</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Elisa Cirillo</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Susan L. Coort</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Daniela Digles</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Friederike Ehrhart</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Pieter Giesbertz</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Marianthi Kalafati</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Marvin Martens</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Ryan Miller</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Kozo Nishida</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Linda Rieswijk</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Andra Waagmeester</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Lars M. T. Eijssen</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chris T. Evelo</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Alexander R. Pico</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Egon L. Willighagen</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>WikiPathways (wikipathways.org) captures the collective 
knowledge represented in biological pathways. By providing a database in
 a curated, machine readable way, omics data analysis and visualization 
is enabled. WikiPathways and other pathway databases are used to analyze
 experimental data by research groups in many fields. Due to the open 
and collaborative nature of the WikiPathways platform, our content keeps
 growing and is getting more accurate, making WikiPathways a reliable 
and rich pathway database. Previously, however, the focus was primarily 
on genes and proteins, leaving many metabolites with only limited 
annotation. Recent curation efforts focused on improving the annotation 
of metabolism and metabolic pathways by associating unmapped metabolites
 with database identifiers and providing more detailed interaction 
knowledge. Here, we report the outcomes of the continued growth and 
curation efforts, such as a doubling of the number of annotated 
metabolite nodes in WikiPathways. Furthermore, we introduce an OpenAPI 
documentation of our web services and the FAIR (Findable, Accessible, 
Interoperable and Reusable) annotation of resources to increase the 
interoperability of the knowledge encoded in these pathways and 
experimental omics data. New search options, monthly downloads, more 
links to metabolite databases, and new portals make pathway knowledge 
more effortlessly accessible to individual researchers and research 
communities.</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2018-01-04</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>eng</td>
					</tr>
					<tr>
					<th>Short Title</th>
						<td>WikiPathways</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>PubMed</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>PMID: 29136241
PMCID: PMC5753270</td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>46</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>D661-D667</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Nucleic Acids Research</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1093/nar/gkx1064">10.1093/nar/gkx1064</a></td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>D1</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>Nucleic Acids Res</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1362-4962</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>6/25/2025, 11:44:18 AM</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>6/25/2025, 11:44:18 AM</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>Humans</li>
					<li>Animals</li>
					<li>Data Curation</li>
					<li>Data Mining</li>
					<li>Databases, Chemical</li>
					<li>Databases, Genetic</li>
					<li>Metabolic Networks and Pathways</li>
					<li>Metabolomics</li>
					<li>Quality Control</li>
					<li>Search Engine</li>
					<li>Software</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_57GV7QWA">Full Text					</li>
					<li id="item_MK5U8QG9">PubMed entry					</li>
				</ul>
			</li>

		</ul>
	
</body></html>